src/components/Events/EventDetail/EventDetail.tsx

```diff
@@ -1,24 +1,40 @@
+import ClayLoadingIndicator from '@clayui/loading-indicator'
 import { useQuery } from '@tanstack/react-query'
 import fetchEvent from 'api/fetch-event'
+import { ClayLoadingIndicatorWrapperStyled } from 'assets/styles/containers'
 const EventDetail = ({ eventId }: IProps) => {
-  const eventDetail = useQuery(['event'], () => fetchEvent({ id: eventId }))
-  const targetString: string =
-    'Podréis asistir en directo por streaming en [youtube](https://www.youtube.com/watch?v=8BGfDJ1j5-Q).\n\nEn la [página del evento ](https://lugspain.github.io/mesdeqa/) podéis consultar más información sobre las sesiones, las comunidades y los ponentes que participan.'
-  const rExp: RegExp = /https:\/\/www\.youtube\.com\/watch\?v=.{11}/
-  const result: RegExpMatchArray | null = targetString.match(rExp)
+  const { status, data } = useQuery(['event'], () =>
+    fetchEvent({ id: eventId })
+  )
+
+  if (data) {
```

Comment:
I'll rather check for `!data` an return `null` in that case
And if data exists, I'll continue with the normal flow of the component.